src/compute/runtime/app/controller/vm.ts

```diff
@@ -0,0 +1,189 @@
+//@ts-nocheck
```

Comment:
Let's try to avoid disabling TypeScript...

Reply:
I was eager to verify, so I banned it and forgot to open it